packages/cardano-services/test/StakePool/TypeormStakePoolProvider/TypeormStakePoolProvider.test.ts

```diff
@@ -264,6 +264,27 @@ describe('TypeormStakePoolProvider', () => {
           expect(response.totalResultCount).toEqual(2);
         });
 
+        it('response is an array of stake pools excluding delisted pool', async () => {
```

Comment:
How does this test work? It seems like it's querying 2 pools by id, then expects 2 pools in the response. Doesn't seem to be asserting exclusion of delisted pools :thinking: